docker/src/lodocker/run_container.py

```diff
@@ -15,6 +15,8 @@ class RunContainer:
     def __init__(self):
         self.shared_doc_dir = "parts"
         self.git_root = Path(__file__).resolve().parents[3]
+        if not Path(self.git_root, '.git').is_dir():
+            self.git_root = Path(__file__).resolve().parents[5]
```

Comment:
Maybe we need a more general approach: What if the user installs the scrips globally instead of inside a venv?

Reply:
What about:

```diff
diff --git a/docker/src/lodocker/run_container.py b/docker/src/lodocker/run_container.py
index 0a71583..1a510b0 100644
--- a/docker/src/lodocker/run_container.py
+++ b/docker/src/lodocker/run_container.py
@@ -14,7 +14,7 @@ from lodocker.helpers import Helpers
 class RunContainer:
     def __init__(self):
         self.shared_doc_dir = "parts"
-        self.git_root = Path(__file__).resolve().parents[3]
+        self.git_root = self.find_git_root()
         logging.info(f"git_root: {self.git_root}")
         self.document_dir = self.git_root / self.shared_doc_dir
         # The home directory of the user in the Docker container
@@ -29,6 +29,20 @@ class RunContainer:
     def doc_mount_string(self) -> str:
         return f"{self.document_dir}:{self.docker_home}/{self.shared_doc_dir}"
 
+    def find_git_root(self) -> Path:
+        # First check if environment variable OPM_REFERENCE_MANUAL_ROOT is set
+        if git_root := os.getenv("OPM_REFERENCE_MANUAL_ROOT"):
+            return Path(git_root)
+        # Assume we are inside the git repository when running this script
+        # Search in the parent directories for the .git directory that also contains
+        # a "parts" directory with a main.fodt file inside it
+        cur_dir = Path.cwd()
+        while cur_dir != "/":
+            if (Path(".git").exists() and Path("parts").exists() and Path("parts/main.fodt").exists()):
+                return Path(cur_dir)
+            cur_dir = cur_dir.parent
+        raise ValueError("Could not find the root of the git repository.")
+
     def font_mount_string(self) -> str:
         return f"{self.font_dir}:{self.docker_font_dir}:ro"
```